milatools/cli/remote.py

```diff
@@ -78,11 +79,16 @@ def get_first_node_name(node_names_out: str) -> str:
 class Remote:
     def __init__(self, hostname, connection=None, transforms=(), keepalive=60):
         self.hostname = hostname
-        if connection is None:
-            connection = Connection(hostname)
-            if keepalive:
-                connection.open()
-                connection.transport.set_keepalive(keepalive)
+        try:
+            if connection is None:
+                connection = Connection(hostname)
+                if keepalive:
+                    connection.open()
+                    connection.transport.set_keepalive(keepalive)
+        except paramiko.SSHException as err:
+            raise SSHConnectionError(node_hostname=self.hostname, error=err)
+        except Exception as err:
+            raise err
```

Comment:
This is unnecessary, exceptions that are not caught by one of the patterns above will be raised by default.

```suggestion
```